docs/pages/admin-guides/management/security/selinux.mdx

```diff
@@ -0,0 +1,198 @@
+---
+title: "SELinux Module for Teleport SSH"
+description: "How to configure SELinux to enforce Teleport SSH."
+---
+
+<Admonition type="note">
+
+This feature is currently a preview feature and is in active development. It will be stable
+in a future release of Teleport.
```

Comment:
Instead of saying "it will be stable" which implies it is currently unstable, I would gave examples of things that are coming in future releases. For example:

```suggestion
This feature is currently under active development. Future releases will extend SELinux policy to support
more Teleport services and features, and integration with Teleport Managed Updates.
```